### ISSUE TITLE
Serializing AutoSklearnClassifier

### ISSUE THREAD

**Issue by clemense** [automl/auto-sklearn#138]
What's the recommended way to serialize an AutoSklearnClassifier?
Pickle doesn't work due to "TypeError: cannot serialize '_io.TextIOWrapper' object".

Thanks

**Comment by mfeurer** [comment]
Pickle does not yet work on the master branch, we have recently fixed it on the development branch though. You can also have a look at #5 and #30.

**Comment by lbustelo** [comment]
Pickle is broken again on development branch and master. It worked fine on b6c3f37289c49eddb34625d92f16b3d1e397cb23, but right after that, it started to fail. I'm getting error

```
PicklingError: Can't pickle <class 'smac.runhistory.runhistory.RunValue'>: attribute lookup RunValue on smac.runhistory.runhistory failed
```

**Comment by mfeurer** [comment]
Sorry for that bug, I'll have a look and try to do a fix as quick as possible.

**Comment by mfeurer** [comment]
This is actually fixed since the release of 0.1.1.
